src/orchestrator/core/orion.py

```diff
@@ -309,13 +176,18 @@ def getListSubscriptions(self,
             if isinstance(json_body_response, list):
                 for subscription in json_body_response:
                     for entity in subscription['subject']['entities']:
+                        # created using NGSIv1
```

Comment:
Not sure of understanding the "created using NGSIv1" vs. "created using NGSIv2" distinction... The objects in subjects.entities has always the same structure, no matter if the entities were created using NGSIv1 or NGSIv2.

We can discuss at skype if you prefer so.

Reply:
fixed in https://github.com/telefonicaid/orchestrator/pull/233/commits/d632c00534ac47af04c56817539fb73dfd7a3aec